workouts/workoutsTest.spec.js

```diff
@@ -0,0 +1,29 @@
+const request = require('supertest');
+const server = require('../index');
+
+let token;
+describe('Workouts', () => {
+  beforeAll(done => {
+    request(server)
+      .post('/auth/login')
+      .send({
+        email: 'foobaz@gmail.com',
+        password: '123456',
+      })
```

Comment:
This user will only get logged in if the user exists. I suggest to register the user instead and clean up after running all tests.